tests/testthat/test-merge_sce_list.R

```diff
@@ -682,7 +702,7 @@ test_that("merging SCEs where 1 altExp is missing works as expected, with altexp
   counts_mat <- counts(merged_altexp)
   sce4_counts <- counts_mat[, merged_sce[[batch_column]] == "sce4"]
   expect_true(
-    all(is.na(sce4_counts))
+    all(sce4_counts == 0)
   )
```

Comment:
I still had this tab open, and thought that here (and below) we can now also use a more specific test:
```
expect_equal(sum(sce4_counts), 0)
```

Reply:
I guess that theoretically that could accidentally pass if you had balanced positives and negatives, so maybe a bit safer:

```
expect_equal(sum(abs(sce4_counts)), 0)
```